Don't present different default plans as guaranteed (#2986)
clerkinc/clerk-docs: docs/guides/billing/default-plans.mdx

## Is it possible to change the default plan?

> Not yet, but we plan to add features in the future to support the paid customers only use case.

Reviewer: ```suggestion
Not yet, but this is planned for a future release to support paid-customers-only use cases.
```
